Sources/BraveWallet/Crypto/Portfolio/AddCustomAssetView.swift

```diff
@@ -9,23 +9,61 @@ import Strings
 import BraveUI
 
 struct AddCustomAssetView: View {
+  @ObservedObject var networkStore: NetworkStore
+  var keyringStore: KeyringStore
   @ObservedObject var userAssetStore: UserAssetsStore
   @Environment(\.presentationMode) @Binding private var presentationMode
+  
+  @ObservedObject private var networkSelectionStore: NetworkSelectionStore
 
   @State private var nameInput = ""
   @State private var addressInput = ""
   @State private var symbolInput = ""
   @State private var decimalsInput = ""
+  @State private var logo = ""
+  @State private var coingeckoId = ""
   @State private var showError = false
+  @State private var showAdvanced = false
+  @State private var isPresentingNetworkSelection = false
+  
+  init(
+    networkStore: NetworkStore,
+    keyringStore: KeyringStore,
+    userAssetStore: UserAssetsStore
+  ) {
+    self.networkStore = networkStore
+    self.keyringStore = keyringStore
+    self.userAssetStore = userAssetStore
+    self.networkSelectionStore = .init(mode: .formSelection, networkStore: networkStore)
+  }
 
   var body: some View {
     NavigationView {
       Form {
+        Section(
+          header: WalletListHeaderView(title: Text(Strings.Wallet.customTokenNetworkHeader))
+        ) {
+          HStack {
+            Button(action: {
+              isPresentingNetworkSelection = true
+            }) {
+              Text(networkSelectionStore.networkSelectionInForm?.chainName ?? Strings.Wallet.customTokenNetworkButtonTitle)
+                .foregroundColor(networkSelectionStore.networkSelectionInForm == nil ? .secondary : Color(.braveLabel))
+            }
+            .disabled(userAssetStore.isSearchingToken)
+            Spacer()
+            Image(systemName: "chevron.down.circle")
+              .foregroundColor(.gray)
+          }
+          .listRowBackground(Color(.secondaryBraveGroupedBackground))
+        }
         Section(
           header: WalletListHeaderView(title: Text(Strings.Wallet.tokenName))
         ) {
           HStack {
-            TextField(Strings.Wallet.enterTokenName, text: $nameInput)
+            TextField("", text: $nameInput)
```

Comment:
Why did we remove the placeholders from all of these fields?

Reply:
aligning with desktop design. plus each field has a header title which has very similar copy.

Reply:
I don't think we should here personally, the view seems a bit blank without them. We use placeholders for most of the other TextField's and we have a placeholder for 'Select Network' row now but none of the other rows. Desktop also has a cursor which changes to a carat over the field to show text entry... I think I'd prefer to keep the placeholders on iOS 😅